static/page_data/rulebook/magic/magic.md

```diff
@@ -4,19 +4,19 @@ The world of Karst is a magical one. Characters can learn to draw power from or
 
 The cosmology of Karst has spawned four distinct types, or Forms, of magic, Thaumaturgy, Theurgy, Spiritualism, and Deceit. A Practitioner of Magic is the term for anyone who has the Knack for one or more of these Forms. Becoming versed in any of these mysteries is possible for any character, but may prove incredibly difficult or involve sacrifice and/or devotion. 
 
-## [Deceit](Deceit)
+## [Deceit](deceit)
```

Comment:
We could make these not-case-sensitive but this makes the links work for now.

Reply:
They "work" but they don't work the same way that react-router works.  They force the whole application to refresh, API to be re-queried, and it also causes a "page %s not found" to flash across the screen although it eventually resolves.

There must be an MDX way to make these react-router Links?

Reply:
I think if we just write them as react-router <Link>s in the MDX and add the Link component via a provider (I saw something about this in the MDax docs) then they should work. I’ll try it out in a bit